sdk/sdk.go

```diff
@@ -25,6 +25,13 @@ type ApplicationLister interface {
 	GetApplication(ctx context.Context, appAddress string) (apptypes.Application, error)
 }
 
+// HeightClient is used to fetch the latest block height.
+//
+// In the future this could be scrapped in favor of passing the latest block height directly to functions that need it.
```

Comment:
```suggestion
// TODO_OPTIONAL_IMPROVE: In the future this could be scrapped in favor of passing the latest block height directly to functions that need it.
```

Reply:
Thanks, this interface is being dropped in the PR update I will be pushing shortly.